packages/cli/package.json

```diff
@@ -37,7 +37,7 @@
     "typescript": "^4.1.2",
     "validator": "^13.6.0",
     "yamljs": "^0.3.0",
-    "yargs": "^14.0.0"
+    "yargs": "^15.3.0"
```

Comment:
Not sure about compatibility, but 17.2.1 is the latest version of yargs.  15.3.0 is about 2 years old.  15.4.1 is about 1 year old.

Reply:
I can make that adjustment to `15.4.1` let me validate